lib/lita/handlers/jenkins.rb

```diff
@@ -85,14 +77,11 @@ def headers
       end
 
       def jobs
-        if self.class.jobs.empty?
-          path = "#{config.url}/api/json"
-          api_response = http.get(path) do |req|
-            req.headers = headers
-          end
-          self.class.jobs = JSON.parse(api_response.body)["jobs"]
+        path = "#{config.url}/api/json"
```

Comment:
Looks great. Not memoizing sure cleans up the code nicely. 

One suggestion I'd make is to store the api url somewhere. Right now we construct the url inside `jobs` but it really should be done (or stored) somewhere else. In SOLID terms this method has too many responsibilities. It's doing two things 1) constructing api url 2) querying the url.

Anyway, just a thought.